src/lib.rs

```diff
@@ -273,6 +274,15 @@ impl Table {
 	pub fn column_iter_mut(&mut self, column: usize) -> ColumnIterMut {
 		return ColumnIterMut(self.rows.iter_mut(), column);
 	}
+
+    pub fn row_iter<'a>(&'a self) -> Iter<'a, Row> {
+        self.rows.iter()
+    }
+
+    pub fn row_iter_mut<'a>(&'a mut self) -> IterMut<'a, Row> {
+        self.rows.iter_mut()
+    }
+
```

Comment:
What about implementing `IntoIterator`? It would make possible to write something like
```rust
for row in &table {
...
}
```